src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/Extensions/ReleaseAmendmentExtensionsTests.cs

```diff
@@ -515,6 +515,64 @@ public void CreateAmendment_ClonesContentBlocks_SameAsBlocksInContent()
             Assert.Equal(block2, contentSection1Block2);
         }
 
+        [Fact]
+        public void CreateAmendment_CopiesKeyStatistics()
+        {
+            var dataBlock = new DataBlock();
+            var release = new Release
+            {
+                Id = Guid.NewGuid(),
+                Version = 1,
+                Published = DateTime.Parse("2020-10-10T13:00:00"),
+                PublishScheduled = DateTime.Parse("2020-10-09T12:00:00"),
+                KeyStatistics = new List<KeyStatistic>
+                {
+                    new KeyStatisticText { Id = Guid.NewGuid() },
```

Comment:
I'd multiline this just to keep with the general style of creating new instances elsewhere in the codebase

Reply:
Changed. I've been doing this kind of one-liner in an effort to reduce the number of lines in tests.